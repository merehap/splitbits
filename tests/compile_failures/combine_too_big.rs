use splitbits::*;

fn main() {
    let a: u16 = 0b1010_0101_0000_1111;
    combinebits!("aaaa aaaa");
}
