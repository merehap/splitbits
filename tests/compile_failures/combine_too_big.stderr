error[E0277]: the trait bound `u8: From<u16>` is not satisfied
 --> tests/compile_failures/combine_too_big.rs:5:5
  |
5 |     combinebits!("aaaa aaaa");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `From<u16>` is not implemented for `u8`
  |
  = help: the following other types implement trait `From<T>`:
            `u8` implements `From<Char>`
            `u8` implements `From<bool>`
  = note: this error originates in the macro `combinebits` (in Nightly builds, run with -Z macro-backtrace for more info)
